src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/HiViUap.java

```diff
@@ -98,6 +101,10 @@ public int getVideoWidthMobile() {
     return driver.findElement(mobileVideoElement).getSize().width;
   }
 
+  public boolean isMobileVideoElementVisible() {
+    return driver.findElement(mobileVideoElement).isDisplayed();
```

Comment:
Interesting. What's the difference between `isDisplayed()` and `isVisible()`? If you don't want to write, I'll google it ;) Although, knowledge share here could be useful for others :japanese_ogre:

Reply:
There is no difference this is the same method, I learned that `org.openqa.selenium.support.ui.ExpectedConditions#elementIfVisible` has inside `isDisplayed` :)

Reply:
Oh, cool. Thanks, I've learnt something more :-) And as we discussed offline yesterday: our Porvata likely handles touch events emulation batter because it's listening to click events not touch events as JWPlayer in mobile version ;-) That's why there are no issues with the test stability as there were with mute/unmute test for JW Player on mobile ([ADEN-6264](https://wikia-inc.atlassian.net/browse/ADEN-6264)) .